### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of https://github.com/gtk-rs/sys/pull/152

cc @sdroege @EPashkin

### CODE REVIEW

**Comment by EPashkin** [comment]
👍

**Comment by sdroege** [comment]
Please also fix the prelude and glib re-exports :)

File: src/auto/object.rs [hunk 1]
```diff
@@ -38,7 +38,8 @@ pub const NONE_OBJECT: Option<&Object> = None;
 pub trait AtkObjectExt: 'static {
     fn add_relationship<P: IsA<Object>>(&self, relationship: RelationType, target: &P) -> bool;
 
-    //fn connect_property_change_handler(&self, handler: /*Unimplemented*/Fn(&Object, /*Ignored*/PropertyValues)) -> u32;
```
Comment (EPashkin) [inline src/auto/object.rs]: This was deprecated since 2.12, and minimal 2.14 now

Comment (GuillaumeGomez) [reply src/auto/object.rs]: So I guess it's all good, no?

**Comment by GuillaumeGomez** [comment]
@sdroege What's wrong in the prelude and which glib reexports are referring to? I didn't find any glib reexport in here so not sure I have the same thing in mind as you do...

**Comment by sdroege** [comment]
> @sdroege What's wrong in the prelude and which glib reexports are referring to? I didn't find any glib reexport in here so not sure I have the same thing in mind as you do...

The prelude should re-export `glib::prelude::*`. And `lib.rs` should **not** `pub use prelude::*`.

File: src/auto/object.rs (hunk 1 above)
Comment (EPashkin) [reply src/auto/object.rs]: Yes. just to prevent possible questions

**Comment by sdroege** [comment]
lgtm

**Comment by GuillaumeGomez** [comment]
Spurious failure merging.

File: src/lib.rs [hunk 2]
```diff
@@ -39,8 +39,6 @@ pub use auto::*;
 
 pub mod prelude;
 
-pub use prelude::*;
```
Comment (sdroege) [inline src/lib.rs]: Yeah so this was stupid :) Please revert that again, thanks!

**Pull request merged**